completion_aggregator/models.py

```diff
@@ -203,7 +203,7 @@ class Aggregator(TimeStampedModel):
     They are considered 100% complete when all descendant blocks are complete.
     """
 
-    user = models.ForeignKey(User)
+    user = models.ForeignKey(User, on_delete=models.DO_NOTHING)
```

Comment:
I think it should cascade. Without the user data this is useless.

Reply:
Changed to CASCADE